src/multi-selector.html

```diff
@@ -5,6 +5,7 @@
 		<div class="cp-multi-selector__pill cps-white cps-bg-gray-10" ng-repeat="item in selectedItems | orderBy: 'display_name'">
 			{{item.display_name}}<i ng-click="removeItem(item)" class="cps-icon cps-icon-sm-close"></i>
 		</div>
+		<span ng-show="inputPlaceholder && (!selectedItems || !selectedItems.length)" class="cp-multi-selector__placeholder">{{inputPlaceholder}}</span>
```

Comment:
At the risk of asking a silly question, why are we adding a span placeholder instead of putting a `placeholder` attribute on the `<input>` element above?